RELEASE_NOTES.md

```diff
@@ -8,10 +8,75 @@ This document contains release notes for the current major release and all patch
+
+### Chef EULA
+
+Chef Infra Server requires an EULA to be accepted by users before it can be installed. Users can accept the EULA in a variety of ways:
+
+- `chef-server-ctl reconfigure --chef-license accept`
+- `chef-server-ctl reconfigure --chef-license accept-no-persist`
+- `CHEF_LICENSE="accept" chef-server-ctl reconfigure`
+- `CHEF_LICENSE="accept-no-persist" chef-server-ctl reconfigure`
+
+Finally, if users run `chef-server-ctl reconfigure` without any of these options, they will receive an interactive prompt asking for license acceptance. If the license is accepted, a marker file will be written to the filesystem unless `accept-no-persist` is specified. Once this marker file is persisted, users no longer need to set any of these flags.
+
+See our [Frequently Asked Questions document](https://www.chef.io/bmc-faq/) for more information on the EULA and license acceptance.
+
+### Deprecation notice
+
+- Deprecated PowerPC and s390x platforms. (https://blog.chef.io/2018/11/01/end-of-life-announcement-for-chef-server-for-linux-on-ibm-z-and-linux-on-ibm-power-systems/)
+- Deprecated Keepalived/DRBD based HA. (https://blog.chef.io/2018/10/02/end-of-life-announcement-for-drbd-based-ha-support-in-chef-server/)
+- Deprecated Ubuntu 14.04 support. (Ubuntu 14 was EoL’d at the end of April 2019)
+
+### Updates and Improvements
+- Updated OpenResty to 1.13.6.2
+    - This fixes two CVEs CVE-2018-9230 and CVE-2017-7529.
+    - This version cannot be built on PowerPC and s390x because those platforms are not supported in mainline luajit.
+- Updated Ruby version to 2.5.5
+- Updated Chef Infra Client to 14.11.21
+- Updated runit to 5.1.1
+- Migrate unit tests from Travis to Buildkite. Reorganize them for improved speed, stability and portability.
+- Habitat:
+    - Added some Habitat packaging improvements with parameterized search_server.
+- Erchef request size increased from 1,000,000 to 2,000,000 bytes to better support inspec scanning via the audit cookbook.
+- Nginx logs cleanup. No longer log 404's in the nginx error log since in the Chef API, 404s are typically not errors. Rather they are often the expected response about an object that doesn't exist. However the logs will continue to show 404's in the request logs.
+- Rendering profiles and data-collector upstreams correctly if their root_url is configured. If the data_collector token secret is not set, 401 response code and an error message will be seen instead of 404.
+
+## 12.19.31 (2019-03-07)
+
+This release contains some Habitat improvements and updates like Openssl in
+Habitat and Omnibus release was done to keep the two in sync.
+
+- chef-server-ctl leverages HAB_LISTEN_CTL envvar if available
+
+
+## 12.19.26 (2019-01-31)
+
+This release contains some minor improvements and updates to included software.
```

Comment:
```suggestion
This release contains some minor improvements and updates to included software:
```

Reply:
I am not sure if "include" (or "included") is the accurate term to use. I need to think about what my suggested word choice would be.